pkg/config/shared.go

```diff
@@ -118,14 +118,20 @@ type HTTP struct {
 
 // Redis represents Redis configuration.
 type Redis struct {
-	Address   string   `name:"address" description:"Address of the Redis server"`
-	Password  string   `name:"password" description:"Password of the Redis server"`
-	Database  int      `name:"database" description:"Redis database to use"`
-	Namespace []string `name:"namespace" description:"Namespace for Redis keys"`
+	Address            string   `name:"address" description:"Address of the Redis server"`
+	Password           string   `name:"password" description:"Password of the Redis server"`
+	Database           int      `name:"database" description:"Redis database to use"`
+	Namespace          []string `name:"namespace" description:"Namespace for Redis keys"`
+	EnableFailover     bool     `name:"enable-failover" description:"Enable failover using Redis Sentinel (default false)"`
```

Comment:
For consistency with the other options, it would be nice to use

`failover.enable`
`failover.addresses`
`failover.master-name`

in the `name` struct tags.

Reply:
In fact, wrapping the failover config into a separate struct and having it as field on `Redis` is a better approach here IMO.
Also I don't think we specify defaults in the description anywhere else?
```suggestion
	Failover struct{
	   Enable     bool `name:"enable" description:"Enable failover using Redis Sentinel"`
	   Addresses  []string `name:"addresses" description:"Redis Sentinel server addresses"`
	   MasterName string   `name:"master-name" description:"Redis Sentinel master name"`
	} `name:"failover"`
```
(or `type RedisFailover struct`)

Reply:
Added options in a new `type RedisFailover struct`. The options are now `--redis.failover.enable`, etc, like @rvolosatovs suggested.